services/scanner/agentpool/agent_pool.go

```diff
@@ -280,39 +283,67 @@ func (ap *AgentPool) SendEvaluateAlertRequest(req *protocol.EvaluateAlertRequest
 	}
 
 	var metricsList []*protocol.AgentMetric
+
```

Comment:
there is no pool-dispatching anymore because combiner subscriptions are 1:1 now. So it finds the "target" bot instead of dispatching to all subscribing bots. Rest is same